core/errors.go

```diff
@@ -29,6 +30,8 @@ func isTemporary(info httperror.Info, err error) bool {
 		return true
 	case "CH001": // request timed out
 		return true
+	case "CH012": // conflict
+		return true
```

Comment:
I guess that wouldn't be sufficient for /configure, because the resulting Config proto isn't deterministic based on the request. The /configure handler will populate with a random core ID, the current time, etc.